scipy/stats/meson.build

```diff
@@ -54,163 +57,176 @@ _stats_c = custom_target('_stats_c',
     _stats_pxd[0],  # __init__.py
     cython_special[7],  # cython_special.pxd
   ],
-  command: [cython, '-3', '--fast-fail',
-            '@INPUT0@', '-o', '@OUTPUT@', '-I', '@BUILD_ROOT@'],
+  command: [
+    cython, '-3', '--fast-fail',
+    '@INPUT0@', '-o', '@OUTPUT@', '-I', '@BUILD_ROOT@'
+  ]
 )
 
 _stats = py3.extension_module('_stats',
   _stats_c,
   c_args: '-Wno-cpp',  # Replace with numpy_nodepr_api after Cython 3.0 is out
-  dependencies : [py3_dep],
-  include_directories : [inc_np],
-  install : true,
-  subdir : 'scipy/stats'
+  dependencies: [py3_dep],
+  include_directories: [inc_np],
+  install: true,
+  subdir: 'scipy/stats'
 )
 
 mvn_module = custom_target('mvn_module',
-  output : ['mvn-f2pywrappers.f', 'mvnmodule.c'],
-  input : 'mvn.pyf',
-  command : [py3, generate_f2pymod, '@INPUT@', '-o', '@OUTDIR@'],
+  output: ['mvn-f2pywrappers.f', 'mvnmodule.c'],
+  input: 'mvn.pyf',
+  command: [py3, generate_f2pymod, '@INPUT@', '-o', '@OUTDIR@']
 )
 
 mvn = py3.extension_module('mvn',
   [mvn_module, 'mvndst.f', fortranobject_c],
   c_args: numpy_nodepr_api,
   fortran_args: fortran_ignore_warnings,
-  dependencies : [py3_dep],
-  include_directories : [inc_np, inc_f2py],
-  install : true,
-  link_language : 'fortran',
-  subdir : 'scipy/stats')
+  dependencies: [py3_dep],
+  include_directories: [inc_np, inc_f2py],
+  install: true,
+  link_language: 'fortran',
+  subdir: 'scipy/stats'
+)
 
 _sobol = py3.extension_module('_sobol',
   ['_sobol.pyx'],
   c_args: numpy_nodepr_api,
-  dependencies : [py3_dep],
-  include_directories : [inc_np],
-  install : true,
-  subdir : 'scipy/stats')
+  dependencies: [py3_dep],
+  include_directories: [inc_np],
+  install: true,
+  subdir: 'scipy/stats'
+)
 
-py3.install_sources(
-  ['_sobol_direction_numbers.npz'],
-  pure : false,
-  subdir : 'scipy/stats'
+py3.install_sources([
+    '_sobol_direction_numbers.npz'
+  ],
```

Comment:
This looks like it should have remained on a single line, because that fits. But well, minor thing.